src/CoreWCF.MSMQ/src/CoreWCF/Channels/MsmqTransportBindingElement.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using CoreWCF.Configuration;
+using CoreWCF.Queue.Common;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace CoreWCF.Channels
+{
+    public sealed class MsmqTransportBindingElement : MsmqBindingElementBase
+    {
+        private int _maxPoolSize = MsmqDefaults.MaxPoolSize;
+        private bool _useActiveDirectory = MsmqDefaults.UseActiveDirectory;
+
+        public MsmqTransportBindingElement() { }
+
+        private MsmqTransportBindingElement(MsmqTransportBindingElement elementToBeCloned)
+            : base(elementToBeCloned)
+        {
+            _useActiveDirectory = elementToBeCloned._useActiveDirectory;
+            _maxPoolSize = elementToBeCloned._maxPoolSize;
+        }
+
+
+        public int MaxPoolSize
+        {
```

Comment:
It doesn't look like you use this property anywhere. This value is used to configure a pool of reusable objects. It's not needed with the current implementation.